setup/product_form_sale_link/odoo/addons/product_form_sale_link

```diff
@@ -0,0 +1 @@
+../../../../product_form_sale_link
```

Comment:
The merge issue is because this should be a symlink, not a regular file?
To be investigated is why the bot does not report the error.

Reply:
No, that one is really a symbolic link: 
![image](https://github.com/OCA/sale-workflow/assets/692075/d4bccddd-62a3-43c8-92da-d4fe93c1d2ed)
You need to delete the file and re-run `pre-commit` locally on a platform that supports symbolic links to generate it.